src/Components/Alert/Components/Form/AlertNameInput.tsx

```diff
@@ -1,26 +0,0 @@
-import { FC } from "react"
-import { Input } from "@artsy/palette"
-import { useFormikContext } from "formik"
-import { AlertFormikValues } from "Components/Alert/Components/Steps/Details"
-import { useAlertContext } from "Components/Alert/Hooks/useAlertContext"
-
-export const AlertNameInput: FC = () => {
-  const { state } = useAlertContext()
-
-  const { values, errors, handleChange, handleBlur } = useFormikContext<
-    AlertFormikValues
-  >()
-
-  return (
-    <Input
-      title="Alert Name"
-      name="name"
-      placeholder={state?.preview?.displayName}
```

Comment:
**suggestion**: We should also remove `displayName` for the preview query as we're no longer have a UI to display it (other than the list view which is separate)